documentation/USAGE.md

```diff
@@ -8,6 +8,9 @@ For now this bundle allows you to configure some properties of a Site Access.
    - Select a **type** and a **theme**, those 2 will be joined with `_` to define the `design`
    - Example: `event` `dark` will setup the design `event_dark` for this Site Access. 
 - **languages**: you can setup the prioritize list of languages
+- **modele**: select content type for home page.
```

Comment:
so no,  that is not true in this PR.
This is a flag, then nobody selects anything.

We must be clear and reflect the business logic.

Reply:
done